### PR TITLE
OLS-598: Data collection scripts in separate directory

### PR DESCRIPTION
**Pull request by tisnik**
## Description

- Data collection scripts in separate directory
- Added documentation to existing scripts

## Type of change

- [x] Refactor
- [ ] New feature
- [ ] Bug fix
- [ ] CVE fix
- [ ] Optimization
- [x] Documentation Update
- [ ] Configuration Update
- [ ] Bump-up dependent library

## Related Tickets & Documents

- Related Issue #[OLS-598](https://issues.redhat.com//browse/OLS-598)
- Closes #

### CODE REVIEW

File: scripts/data_collection/README.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
+# Data collection tools
+
+## `feedback_aggregator.py`
```
Comment (onmete) [inline scripts/data_collection/README.md:3]: Maybe rename to user_data_aggregator?

Comment (tisnik) [reply scripts/data_collection/README.md:3]: maybe, ok, will do it

Comment (onmete) [reply scripts/data_collection/README.md]: or other better name, as it is not just about feedback

Comment (tisnik) [reply scripts/data_collection/README.md]: better @onmete ?

**Pull request merged**